src/library/scala/collection/Iterable.scala

```diff
@@ -142,7 +141,16 @@ trait Iterable[+A] extends IterableOnce[A] with IterableOps[A, Iterable, Iterabl
 trait IterableOps[+A, +CC[_], +C] extends Any with IterableOnce[A] with IterableOnceOps[A, CC, C] {
 
   /**
-    * Type alias to `CC`. It is used to provide a default implementation of the `fromSpecific`
+    * Type alias to `CC`. It is used to provide a default implementation of the `iterableFactory`
+    * operation.
+    *
+    * Due to the `@uncheckedVariance` annotation, usage of this type member can be unsound and is
+    * therefore not recommended.
+    */
+  //protected type IterableCC[X] = CC[X] @uncheckedVariance
+
+  /**
+    * Type alias to `C`. It is used to provide a default implementation of the `fromSpecific`
```

Comment:
I think you forgot to remove these lines as well as the next lines (until L159)

Reply:
You're fast! Yes, thank you. Will continue on this tomorrow, unless it's already clear to the reviewers that this road leads nowhere :-)